composer.json

```diff
@@ -15,7 +15,7 @@
     "ext-curl": "*",
     "ext-json": "*",
     "ext-mbstring": "*",
-    "guzzlehttp/guzzle": "^6.3"
+    "guzzlehttp/guzzle": "^6.3|^7.0.1"
```

Comment:
I thought you had to use || for these like `"^6.3 || ^7.0.1"` but if this works then ok for me 😄